### ISSUE TITLE
<main> tag in modal component causes `multiple main landmarks` error in a11y check of a new enduser

### ISSUE THREAD

**Issue by eugenebakaiev-okta** [okta/odyssey#567]
## Detailed description

> Is this issue related to a specific component?

Yes, it's related to a Modal component.

> What did you expect to happen? What happened instead?

I expected to have one `<main>` tag on a page which uses Modal component. I got two `<main>` tags on a page because one of them is located inside Modal component

> What browser & version are you experiencing this in?

Every

> Is there a specific WCAG checkpoint or ARIA guideline being violated?

Aceessibility checker identifies it as `multipleMainLandmarks`
![image](https://user-images.githubusercontent.com/20933838/86949287-18c37680-c157-11ea-8f72-b36a1d844157.png)

> Is this issue related to a specific assistive technology?

Screen readers are affected.

## Screenshots

If relevant, please include screenshots to help illuminate the issue.

## Additional information

More information could be found here: https://oktainc.atlassian.net/browse/OKTA-282892
